cameraViewEx/src/main/api21/com/priyankvasa/android/cameraviewex/Camera2.kt

```diff
@@ -1143,6 +1143,12 @@ internal open class Camera2(
             setVideoSize(videoSize.width, videoSize.height)
             setVideoEncoder(config.videoEncoder.value)
             setAudioEncoder(config.audioEncoder.value)
+
+            // Let's not have videos less than one second
+            if (config.maxDuration >= VideoConfiguration.DEFAULT_MIN_DURATION) {
+                setMaxDuration(config.maxDuration)
+            }
```

Comment:
Here an warning would be better rather than silently not setting the value.
```
if (config.maxDuration >= VideoConfiguration.DEFAULT_MIN_DURATION) {                
    setMaxDuration(config.maxDuration)
} else {
    listener.onCameraError(CameraViewException("${config.maxDuration} is not a valid max duration value for video recording. Using minimum default ${VideoConfiguration.DEFAULT_MIN_DURATION}."))
    setMaxDuration(VideoConfiguration.DEFAULT_MIN_DURATION)
}
```

Reply:
Updated to have the warning

Reply:
Updated to include the warninf